packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -66,6 +66,11 @@ export default new Bundler({
         let assets = bundleGraph.getDependencyAssets(dependency);
         let resolution = bundleGraph.getDependencyResolution(dependency);
 
+        if (bundleGraph.isDependencySkipped(node.value)) {
```

Comment:
Since this is done implicitly in `bundleGraph.addAssetGraph`, should this check be done in `bundleGraph.traverse` itself and the callback never be called?

Reply:
Actually, I've had to add this kind of check to `addAssetGraphToBundle` (so inside `BundleGraph`). But ideally this would be configureable by the caller (regarding https://github.com/parcel-bundler/parcel/pull/4983)

Reply:
> Are there any cases where we would want to traverse over skipped nodes?

Same as https://github.com/parcel-bundler/parcel/pull/4983, it's if the plugin author's definition of "skip" and Parcel's opinion diverge (= there is some other decision factor besides symbols).

I've now added an optional callback to `addAssetGraphToBundle` that can be used to control this. I think we should just keep the explicit check in the bundler traversal.